lib/graphite-api/buffer.rb

```diff
@@ -3,16 +3,16 @@
 # Handle Socket & Client data streams
 # -----------------------------------------------------
 # Usage:
-#     buff = GraphiteAPI::Buffer.new(GraphiteAPI::Utils.default_options)
-#     buff << {:metric => {"load_avg" => 10},:time => Time.now}
-#     buff << {:metric => {"load_avg" => 30},:time => Time.now}
+#     buff = GraphiteAPI::Buffer.new(GraphiteAPI::Client.default_options)
+#     buff << {:metric => {"load_avg" => 10},:time => Time.now, :aggregation_method => :avg}
```

Comment:
default_aggregation_method

Reply:
This is actually `aggregation_method` because it is a specific push